Header header

bool intersect
float32[] center_x    #x-value of middle Collision Area
float32[] center_y    #y-value of middle Collision Area
float32[] radius      #radius of Collision Area
float32[] time      #radius of Collision Area

obstacle_detector/CircleObstacle[] circles  #used obstacles
